Fixed Giant Fly Wing behavior (#6204)
rathena/rathena: doc/script_commands.txt

              attached player (will fail if there's no player attached).
Leader:       All party members are warped to the leader's position. The leader must
              be online and in the current map-server for this to work.

> RandomAll:    All party members are warped to the same random position in their current map

Reviewer: ```suggestion
RandomAll:    All party members are warped to a random position in their current map
```

Author: I phrased it that way to avoid confusion between Random and RandomAll, because in this case all players are warped to the same random cell.
